src/globus_cli/parsing/shared_options/endpointish.py

```diff
@@ -162,29 +186,25 @@ def _apply_endpointish_create_or_update_params(
                 " When set on mapped collections, this policy is inherited by any "
                 "guest collections"
```

Comment:
_Brain nudge: the whitespace in `verify_help` (here, where it's concatenated, and above in its definition) should be standardized at a later date._

Reply:
At one point for gcs-cli, I wrote a series of tests for helptext because there were various tricky issues to run down. Between whitespace and my f-string typos, maybe we'd benefit from something similar here.

Perhaps a suite of tests which invokes `--help` on every command and asserts things about the output. I'll chew on this idea today.